ts/DataGrid/Utils.ts

```diff
@@ -131,8 +132,29 @@ namespace DataGridUtils {
             return '';
         }
     }
-}
 
+    /**
+     * Sets an element's content, checking whether it is HTML or plain text.
+     * Should be used instead of element.innerText when the content can be HTML.
+     * 
+     * @param element
+     * Parent element where the content should be.
+     *
+     * @param content
+     * Content to render.
+     */
+    export function setHTMLContent(
+        element: HTMLElement,
+        content: string
+    ): void {
+        if (isHTML(content)) {
+            const formattedNodes = new AST(content);
+            formattedNodes.addToDOM(element);
+        } else {
+            element.innerText = content;
+        }
+    }
```

Comment:
This is exactly the same piece of code like here: https://github.com/highcharts/highcharts/blob/master/ts/DataGrid/Table/Content/TableCell.ts#L240

Maybe we should extract this method to Utils and use in two places?